### PR TITLE
Release - API Management service: January, 2022

### PR DESCRIPTION
**Pull request by mikebudzynski**

### CODE REVIEW

File: changelogs/api-management-service.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
 
 Developer portal follows an independent release lifecycle and the [per-release changelog is available in the developer portal's GitHub repository](https://aka.ms/apimdevportal/releases).
 
+## Release - API Management service: January, 2022
+
+A regular Azure API Management service update was started on January 20, 2021, and included the following new features, bug fixes, and other improvements. It may take several weeks for your API Management service to receive the update.
```
Comment (solankisamir) [inline changelogs/api-management-service.md:7]: We are in  `2022` :)

Comment (mikebudzynski) [reply changelogs/api-management-service.md:7]: Every year, the same problem: https://github.com/Azure/API-Management/commit/7e913caf468ca4eba8f49af1b28edb836f0c7652

File: changelogs/api-management-service.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+
+### Featured
+
+1. [Managed certificate support is now in public preview](https://azure.microsoft.com/updates/public-preview-managed-certificate-support-for-azure-api-management/).
+
+### New
+
```
Comment (solankisamir) [inline changelogs/api-management-service.md:14]: We added `East Asia` as a new supported region for Availability Zones https://aka.ms/apimaz. Do we announce those here?

Comment (solankisamir) [reply changelogs/api-management-service.md:14]: We also released new management sdk for .NET https://www.nuget.org/packages/Microsoft.Azure.Management.ApiManagement/8.0.0-preview

**Review by solankisamir — APPROVED** [review]

File: changelogs/api-management-service.md [hunk 3]
```diff
@@ -2,6 +2,28 @@
...
+### Fixed
+
+1. Newly created diagnostic settings will no longer be configured to log request query parameters by default. As part of this change, [`Diagnostic` entity](https://docs.microsoft.com/rest/api/apimanagement/current-ga/diagnostic/create-or-update#datamasking)'s `dataMasking.queryParams` properties will be set with the following wildcard configuration `{ "value": "*", "mode": "Hide"}`. The same wildcard configuration can also be applied to `dataMasking.headers`.
+2. Self-hosted gateway now properly handles a certificate change (`certificateId`) for existing hostnames.
+3. Multiple `validate-content` policies can now be specified in a single policy section.
+4. It is now possible to delete a resource group with an `stv2`-based API Management service in a virtual network. Previously, the deletion could fail due to an unreleased public IP resource.
+5. `ConfigurationChange` event is no longer logged in Resource Health for API Management service backups.
```
Comment (solankisamir) [inline changelogs/api-management-service.md:23]: do people come back to you, asking what is resource Health? https://docs.microsoft.com/en-us/azure/service-health/resource-health-overview

File: changelogs/api-management-service.md (hunk 2 above)
Comment (mikebudzynski) [reply changelogs/api-management-service.md:14]: Added

**Comment by solankisamir** [comment]
.NET SDK - https://www.nuget.org/packages/Microsoft.Azure.Management.ApiManagement/8.0.0-preview

File: changelogs/api-management-service.md [hunk 4]
```diff
@@ -2,6 +2,31 @@
...
+### Featured
+
+1. [Managed certificate support is now in public preview](https://azure.microsoft.com/updates/public-preview-managed-certificate-support-for-azure-api-management/).
+
+### New
+
+1. Published developer portals are now automatically upgraded to new portal releases, without the need to republish the portal manually. Automated upgrades will preserve the latest published content; they won't publish saved but unpublished content.
+2. You can now use curly brackets in a SOAP action URL template (for example, `/soapAction={wildcard}`) to define a wildcard SOAP action, which will match any SOAP request that doesn't have a dedicated action defined in the API. The value inside the curly brackets doesn't affect the execution.
+3. Availability zones are now supported in the East Asia region.
+4. New .NET SDK for the management API [is now available](https://www.nuget.org/packages/Microsoft.Azure.Management.ApiManagement/8.0.0-preview).
```
Comment (tomkerkhove) [inline changelogs/api-management-service.md:18]: Let's be a bit more explicit
```suggestion
4. New .NET SDK for the management API [is now available in preview](https://www.nuget.org/packages/Microsoft.Azure.Management.ApiManagement/8.0.0-preview).
```

File: changelogs/api-management-service.md [hunk 5]
```diff
@@ -2,6 +2,31 @@
...
+2. Self-hosted gateway now properly handles a certificate change (`certificateId`) for existing hostnames.
+3. Multiple `validate-content` policies can now be specified in a single policy section.
+4. It is now possible to delete a resource group with an `stv2`-based API Management service in a virtual network. Previously, the deletion could fail due to an unreleased public IP resource.
+5. `ConfigurationChange` event is no longer logged in [Resource Health](https://docs.microsoft.com/azure/service-health/resource-health-overview) for API Management service backups.
+6. `tracestate` header values are no longer truncated after the first key-value pair.
+7. An attempt to deploy an `stv2` API Management service into a virtual network subnet with an `stv1` API Management service will now result in a descriptive error message.
```
Comment (tomkerkhove) [inline changelogs/api-management-service.md:28]: What is an stv2 / stv1 instance?

Comment (ErikMogensen) [reply changelogs/api-management-service.md:28]: > What is an stv2 / stv1 instance?

https://docs.microsoft.com/en-us/azure/api-management/compute-infrastructure

Comment (tomkerkhove) [reply changelogs/api-management-service.md:28]: Exactly, but not everyone knows.

But let's add a link to it indeed, best way.

```suggestion
7. An attempt to deploy an `stv2` API Management service into a virtual network subnet with an `stv1` API Management service will now result in a descriptive error message. (Learn more about our [compute platform](https://docs.microsoft.com/en-us/azure/api-management/compute-infrastructure#compute-platform-versions))
```

**Pull request merged**